dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/security/KeyVault.java

```diff
@@ -152,16 +152,23 @@ public static PrivateKey loadPrivateKeyFromJavaEncoding(String privatePart, Stri
 	}
 
 	public static PrivateKey loadPrivateKeyFromPem(String privatePart, String algorithm) {
+		PemReader readerPem = null;
+		PrivateKey result = null;
 		try {
 			var reader = new StringReader(privatePart);
-			var readerPem = new PemReader(reader);
+			readerPem = new PemReader(reader);
 			var obj = readerPem.readPemObject();
-			readerPem.close();
-			return KeyFactory.getInstance(algorithm).generatePrivate(new PKCS8EncodedKeySpec(obj.getContent()));
+			result = KeyFactory.getInstance(algorithm).generatePrivate(new PKCS8EncodedKeySpec(obj.getContent()));
 		} catch (InvalidKeySpecException | NoSuchAlgorithmException | IOException e) {
 			logger.error("Exception loading private key from PEM", e);
-			return null;
+		} finally {
+			try {
+				readerPem.close();
+			} catch (IOException e) {
+					logger.error("Exception closing PEM reader: {}", e.getMessage(), e);
```

Comment:
nitpick: this looks like one tab to many.
(`readerPem` and `logger` should have the same intend, just looking at the brackets)
```suggestion
				logger.error("Exception closing PEM reader: {}", e.getMessage(), e);
```